msbrainpy/atlas/wrangle.py

```diff
@@ -738,3 +738,68 @@ def writeStructDFsandMeans(df, structs, prefix, groupby='section_data_set_id'):
     out.to_csv(saveName, sep='\t')
     return out
 
+
+# ----------------------------------- Get Absolute File and Directory Paths -------------------------------------------
+def get_image_addresses(path, age_id=None):
+    
+    """
+    FUNCTION:Return a dictionary with all of the absolute file paths for images belonging to a gene directory in
+        the format produced by msbrainpy.atlas.query.download_gene_images().
+        Approx. {'entrez_id-<gene id>_<Acronym>' : {'plane_of_section-<1 or 2>' : {'age_id-<age id>_id-<id>' :
+            [<file path 0>, <file path 1>, ...]}
+    :param path: Absolute file path of the directory containing the gene directories (str)
+    :param age_id: age id (15 = P56) (int or None)
+    :return: dictionary of image paths (dict)

```

Comment:
As mentioned above, it's best to use the NumPy docstring formatting guide when working with scientific Python:

https://numpy.org/doc/stable/docs/howto_document.html